src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -108,6 +108,8 @@ public void testPostBinary() throws Exception {
                 CREATED.getStatusCode(), status);
         assertEquals("Location did not match slug", serverAddress + slug, response.getLocation().toString());
 
+        assertNotNull("Didn't find linked description!", response.getLinkHeaders("describedby").get(0));
```

Comment:
I don't want to drag this out, but I thought that the intent of this PR was to support Link headers with three or more parts. I am not seeing this integration test demonstrating that.